draft-ietf-quic-transport.md

```diff
@@ -3418,14 +3418,15 @@ ACK Block (repeated):
 ### Sending ACK Frames
 
 Implementations MUST NOT generate packets that only contain ACK frames in
-response to packets which only contain ACK frames. However, they MUST
-acknowledge packets containing only ACK frames when sending ACK frames in
-response to other packets.  Implementations MUST NOT send more than one packet
-containing only an ACK frame per received packet that contains frames other than
-an ACK frame.  Packets containing non-ACK frames MUST be acknowledged
-immediately or when a delayed ack timer expires. The delayed ack timer MUST
-NOT delay an ACK for longer than an RTT, which ensures an ACK frame is sent
-at least once per RTT if new packets needing acknowledgement were received.
+response to packets which only contain ACK and PADDING frames. However, they
+MUST acknowledge packets containing only ACK and PADDING frames when sending
+ACK frames in response to other packets.  Implementations MUST NOT send more
+than one packet containing only an ACK frame per received packet that contains
```

Comment:
"containing only ACK or PADDING frames" ?

Reply:
To clarify, are you saying I should replace "ACK and PADDING" with "ACK or PADDING" everywhere or another suggestion?

Reply:
Just the one instance here that you only have "ACK".

Reply:
Should we mention that Challenge and Challenge response frames have the same treatment? What about extension frames that would be defined using the extension mechanism?

Reply:
@huitema there is already an issue open on extension frames https://github.com/quicwg/base-drafts/issues/1487

Reply:
I think only an ACK is most correct there.  If you really wanted to send ACK and PADDING, that would count towards bytes in flight, and then congestion control is involved, which provides a different rate limit.

Reply:
SGTM